.azurepipelines/MuDevOpsWrapper.yml

```diff
@@ -85,5 +93,18 @@ jobs:
     vm_image: $(vm_image)
     container_build: ${{ parameters.container_build }}
 
-- ${{ parameters.extra_jobs }}
+- ${{ if eq(parameters.rust_build, true) }}:
+  - job: CargoCmds
+    displayName: Workspace Cargo Commands
 
+    container:
+      image: ghcr.io/microsoft/mu_devops/ubuntu-22-build:9ab29bc

```

Comment:
between this and line 78, we will have two locations to update if we need to change the container image, would it make sense to localize it into a parameter?

Reply:
They're both updated with from the single `linux_build_container` value in [mu_devops/.sync/Version.njk](https://github.com/microsoft/mu_devops/blob/main/.sync/Version.njk) in the file sync so there's no manual maintenance needed directly in the file in this repo.